src/cmd/services/m3collector/config/config.go

```diff
@@ -37,7 +36,7 @@ import (
 type Configuration struct {
 	Metrics       instrument.MetricsConfiguration `yaml:"metrics"`
 	Logging       zap.Config                      `yaml:"logging"`
-	ListenAddress listenaddress.Configuration     `yaml:"listenAddress" validate:"nonzero"`
+	ListenAddress string                          `yaml:"listenAddress" validate:"nonzero"`
```

Comment:
do we actually call validate on this?

Reply:
Doesn't this get auto validated by the yaml parser, given the notation?